### DISCUSSION TITLE
关于4399部分游戏加载卡在0%的解决方法

### DISCUSSION THREAD

**Discussion by Mzying2001** [Mzying2001/CefFlashBrowser#19] [General]
如图，4399部分游戏在加载会卡在0%
![image](https://user-images.githubusercontent.com/41951400/210563107-df07fe93-55ab-43fb-906e-d454c911a7a4.png)

## 解决方式
方法很简单，只需要在开始游戏前选择使用“全屏1”或者“全屏2”模式玩游戏即可
![image](https://user-images.githubusercontent.com/41951400/210563274-be49f2f3-1d72-4d3b-be91-e537ee81c91b.png)
如图，在这种模式下游戏可以正常运行
![image](https://user-images.githubusercontent.com/41951400/210563343-4ac93765-acc9-4e99-baa1-c8920b7f5c03.png)

**Comment by Mzying2001** [comment]
其他网站如果无法加载可以看看浏览器窗口右上角是否有一个感叹号图标，可以在单独的窗口中加载Flash内容。（这个功能原本是针对这个问题搞的，但后来4399为了防止抓包运行swf时会检测环境导致这个方法对4399不适用）
![image](https://user-images.githubusercontent.com/41951400/210565048-bf8ec788-7bf9-4102-a48c-5e918551d0a9.png)

**Comment by HuoYanLongWa** [comment]
我说呢，作者大大真周到没想到唯几个问题里面有我想问的
